### ISSUE TITLE
Add Some Ambience

### ISSUE THREAD

**Issue by wtchappell** [TelepathicGrunt/Bumblezone#14]
Currently the Bumblezone is pretty quiet, and it has no specific ambience. I've been doing some experiments with the AmbientSounds mod to add some background ambience to the Bumblezone, and I think it adds a lot!

I downloaded the audio from this video: https://www.youtube.com/watch?v=jpZR1xDsQiw
And then cut out a random snippet, to create this:  [hivenoise.zip](https://github.com/TelepathicGrunt/Bumblezone/files/4944799/hivenoise.zip)
(It's just an OGG vorbis sound file in a Zip so GitHub would allow me to attach it.)

Then I've got it set up with AmbientSounds as follows:
```	
    {
      "name": "bumblezone",
      "dimension-names": [ "the_bumblezone", "bumblezone" ],
      "biome-selector": {
        "always": false
      },
      "regions": [
        {
          "name": "hive",
          "volume": 0.08,
          "sounds": [
            {
              "name": "hive-ambience",
              "files": [ "ambientsounds:animals.hive-ambience" ],
	      "outside": true
            }
          ]
        },
	{
          "name": "hive-walls",
          "volume": 0.02,
          "sounds": [
            {
              "name": "hive-walls-ambience",
              "files": [ "ambientsounds:animals.hive-ambience" ],
	      "outside": false
            }
          ]
        }
      ]
    }
```

Basically, the sound is much more intense the more 'outside' you are, and gets quieter (but never below 0.02) if you tunnel deep into the walls.

I presume there's a way to integrate this directly into the mod proper, but I don't know enough about the Forge APIs and how other dimensions set up their background ambience to make a pull request myself. If you use to use that particular clip, you'd need to run it by the video uploader before redistributing it first, too: https://www.youtube.com/channel/UCYCmal9NJFAJqPzJXkPkcpQ

(Though there are probably lots of similar videos if permissions are problematic.)

**Comment by TelepathicGrunt** [comment]
The issue with that is the license with using other people's sounds which is why I am hesitate about that. 

The main source of noise in The Bumblezone is the Brood Blocks and Bees. I would had seen about adding ambient sound to the dimension but that is not possible because of this line:
https://github.com/TelepathicGrunt/Bumblezone/blob/78ea6951ceefa01fa9deec509c78a115e3d52213/src/main/java/net/telepathicgrunt/bumblezone/dimension/BzDimension.java#L57

MusicTicker.MusicType is an enum that forge forgot to apply IExtendableEnums to. As of right now, this is the only ambient sound avaliable.
![image](https://user-images.githubusercontent.com/40846040/87887397-c1dd5d00-c9f2-11ea-9115-aa9787b5afb7.png)

There may be other ways to hack in ambient sounds but will be much more trickier than the vanilla way that should've been patched by forge

**Comment by TelepathicGrunt** [comment]
1.16 version has ambiance in the way of loud bee buzzing sounds lmao. The biome json files allowed an entry for ambiance sound so I stuck the vanilla bee sounds into it. 

I'll close this now as I give up trying to add ambiance to 1.15.2 and I no longer support 1.15.2.

Though if people really don't like the current loud bee buzzing sound, they could PR a sound file to me that doesn;t have any legal issues and I'll replace the vanilla bee sound with it for 1.16+. :)
